modules/about-rhde.adoc

```diff
@@ -34,14 +34,14 @@ The latest release notes for each product that is part of {product-title} are av
 [id="device-edge-compatibility_{context}"]
 == {product-title} product release compatibility matrix
 
-{microshift-first} and {op-system} work together as a single solution for device-edge computing. To successfully pair your products, use the verified releases together for each as listed in the following table:
+{op-system-first} and {microshift} work together as a single solution for device-edge computing. Supported configurations of {product-title} use verified releases for each together as listed in the following table:
 
 [cols="4",%autowidth]
 |===
 ^|*{op-system-ostree} version*
 ^|*{microshift} version*
 ^|*{microshift} release status*
-^|*{microshift} supported updates*
+^|*{microshift} supported updates* ^[1]^
```

Comment:
I think adding a footnote on the operating system makes this a bit complicated, esp as we explicitly listen the operting system here, too.
Another thing I dont like is the "-> maximum two minor versions". That could be interpreted as a promise into the future, which we might not be able to hold up. Also, it is not stating that an update from V4.14 and 4.15. is possible.

Proposal:

We should keep it as with the previous versions and rows in this table: we list FROM which versions an update is possible.
Meaning, the 9.4 - 4.16 row could contain in the right "supported updates" cell:
4.14 -> 4.16
4.15 -> 4.16
4.16.0→4.16.z

Reply:
Done